Create SITE.yaml

irodero opened the pull request:
Adding VDC test site 1 for Scientific Computing and Imaging Institute at the University of Utah.

brianhlin commented on topology/University of Utah/Utah-SCI-VDC1/SITE.yaml:
```diff
@@ -0,0 +1,12 @@
+LongName: Scientific Computing and Imaging Institute VDC test site 1.
+Description: University of Utah's Scientific Computing and Imaging Institute 
+  (SCI) located in Salt Lake City. Test site for the Virtual Data Collaboratory
+  (VDC) initiative.
```
```suggestion
LongName: Scientific Computing and Imaging Institute Virtual Data Collaboratory
Description: >
    University of Utah's Scientific Computing and Imaging Institute
    (SCI) located in Salt Lake City. Test site for the Virtual Data
    Collaboratory (VDC) initiative.
```

brianhlin reviewed (commented):
Hi @irodero thanks for the PR! Would you be able to rename the site from `Utah-SCI-VDC1` to `Utah-SCI-VDC`? We generally mark groups of resources under a site for testing instead of the entire site. For example, at the Center for High Throughput Computing (CHTC), we have separate YAML files for our testing resources vs our production ones:

- https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC-ITB.yaml#L3
- https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC.yaml#L4

brianhlin reviewed (approved):
LGTM, thanks for the updates!

brianhlin commented on topology/University of Utah/Utah-SCI-VDC/SITE.yaml:
```diff
@@ -0,0 +1,13 @@
+LongName: Scientific Computing and Imaging Institute Virtual Data Collaboratory
+ Description: >
+     University of Utah's Scientific Computing and Imaging Institute
+     (SCI) located in Salt Lake City. Test site for the Virtual Data
+     Collaboratory (VDC) initiative.
```
```suggestion
Description: >
  University of Utah's Scientific Computing and Imaging Institute
  (SCI) located in Salt Lake City. Test site for the Virtual Data
  Collaboratory (VDC) initiative.
```

Fix spacing

irodero commented on topology/University of Utah/Utah-SCI-VDC1/SITE.yaml:
```diff
@@ -0,0 +1,12 @@
+Description: University of Utah's Scientific Computing and Imaging Institute 
+  (SCI) located in Salt Lake City. Test site for the Virtual Data Collaboratory
+  (VDC) initiative.
+ID: 
```
Thank you very much for processing and fixing the new site request.

irodero commented:
> Hi @irodero thanks for the PR! Would you be able to rename the site from `Utah-SCI-VDC1` to `Utah-SCI-VDC`? We generally mark groups of resources under a site for testing instead of the entire site. For example, at the Center for High Throughput Computing (CHTC), we have separate YAML files for our testing resources vs our production ones:
> 
> * https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC-ITB.yaml#L3
> * https://github.com/brianhlin/topology/blob/project-compat-symlink/topology/University of Wisconsin/CHTC/CHTC.yaml#L4

Thank you, Brian. It does make completely sense. I have updated the directory name and added a resource group for a test origin. Your assistance is greatly appreciated.

irodero commented:
Dear @brianhlin, it looks like the checks are failing. Please let me know if I need to update any of the two yaml files. I left the groupID empty in the resource group yaml file. Thank you very much for your assistance.

brianhlin commented on topology/University of Utah/Utah-SCI-VDC/Utah-SCI-VDC-ITB.yaml:
```diff
@@ -0,0 +1,26 @@
+GroupDescription: Resource group for the SCI VDC ITB site
+GroupID: 
```
```suggestion
GroupID:  1336
```

brianhlin commented on topology/University of Utah/Utah-SCI-VDC/Utah-SCI-VDC-ITB.yaml:
```diff
@@ -0,0 +1,26 @@
+    Active: true
+    Description: >-
+        This is a testing OSDF origin server on a VDC resource.
+    ID: 
```
```suggestion
    ID: 1401
```

brianhlin reviewed (commented):
Adding IDs

brianhlin commented on topology/University of Utah/Utah-SCI-VDC/SITE.yaml:
```diff
@@ -0,0 +1,13 @@
+  University of Utah's Scientific Computing and Imaging Institute
+  (SCI) located in Salt Lake City. Test site for the Virtual Data
+  Collaboratory (VDC) initiative.
+ ID: 10348
+ AddressLine1: 72 S Central Campus Drive, Room 3750
+ City: Salt Lake City
+ Country: United States
+ State: UT
+ Zipcode: '84112'
+ Latitude: 40.7608
+ Longitude: -111.891
```
```suggestion
ID: 10348
AddressLine1: 72 S Central Campus Drive, Room 3750
City: Salt Lake City
Country: United States
State: UT
Zipcode: '84112'
Latitude: 40.7608
Longitude: -111.891
```

irodero commented:
Hi @brianhlin, I believe the checks have passed. Is this ready to be merged? I accidentally closed the PR and re-opened it. Thank you!

brianhlin commented:
Yup, we're all good. Thanks again for the PR!
